src/js/view/floating/floating-controller.ts

```diff
@@ -239,6 +241,10 @@ export default class FloatingController {
         // Perform resize and trigger "float" event responsively to prevent layout thrashing
         this._model.trigger('forceResponsiveListener', {});
     }
+    
+    transitionFloating(isTransitionIn: boolean): void {
```

Comment:
When `isTransitionIn` is `true`, can we signal to the view that we are intentionally hiding the player so that it ignores resizes to `0` until this is complete?

And when `isTransitionIn` is `false` maybe that's when we should have the view run `responsiveListener`